homeassistant/components/daikin/config_flow.py

```diff
@@ -37,10 +37,13 @@ def schema(self):
                 vol.Required(CONF_HOST, default=self.host): str,
                 vol.Optional(CONF_API_KEY): str,
                 vol.Optional(CONF_PASSWORD): str,
+                vol.Required(CONF_ENABLE_OUTDOOR_SENSORS, default=True): bool,
```

Comment:
We shouldn't add an option for this. The user can disable/enable entities and devices that they aren't or are interested in.

Reply:
This is actually a poor wording which I can correct: it's not about disabling/enabling the sensors but rather about not creating them at all. The goal is still the same: avoiding confusion about having multiple times the same sensors in different config entries.

We have three solutions:

1. Keeping the integration as-is and creating N "All devices' energy consumption" for N devices even though they will all report the same value. The list of such sensors is: outdoor temp, compressor freq, all devices' energy & power consumption
2. Not creating the outdoor sensors when the option is unchecked in the config entry (my last commit)
3. Automatically creating a unique "Outdoor config entry" when an indoor one is setup (cf [example](https://github.com/home-assistant/core/blob/0.68.0/homeassistant/components/hue/__init__.py#L104))

Which one do you prefer?

Reply:
@emufan @fredrike Do you have an opinion on the above matter?

Reply:
I don't really know. I guess having multiple sensors with same value would be best (but requires a description in the docs).

Reply:
Ok I've removed the option